src/wwwroot/css/themes/Beweish.css

```diff
@@ -0,0 +1,372 @@
+    box-shadow: var(--shadow-sm);
+}
+
+.card:hover {
+    box-shadow: var(--shadow-md);
+}
+
+/* Input styling */
+input, select, textarea {
+    background-color: var(--background);
+    border: 1px solid var(--border-color);
+    color: var(--text);
+    transition: all 0.3s ease;
+}
+
+input:focus, select:focus, textarea:focus {
+    border-color: var(--emphasis);
+    outline: none;
+    box-shadow: var(--shadow-sm);
+}
+
+/* Link styling */
+a {
+    color: var(--emphasis);
+    transition: color 0.3s ease;
+}
+
+a:hover {
+    color: var(--emphasis-soft);
+}
+
+/* Modal styling */
+.modal-content {
+    background: var(--background-panel);
+    border: 1px solid var(--border-color);
+    box-shadow: var(--shadow-lg);
+}
+
+.modal-header {
+    border-bottom: 1px solid var(--border-color);
+}
+
+.modal-footer {
+    border-top: 1px solid var(--border-color);
+}
+
+/* Progress bar */
+.progress {
+    background: var(--background-gray);
+}
+
+.progress-bar {
+    background: var(--emphasis);
+}
+
+/* Alert styling */
+.alert-success {
+    background: var(--green);
+    color: var(--text);
+}
+
+.alert-warning {
+    background: #ffbb00;
+    color: var(--text);
+}
+
+.alert-danger {
+    background: var(--red);
+    color: var(--text);
+}
+
+.alert-info {
+    background: var(--text-soft);
+    color: var(--background);
+}
+
+#t2i-top-split-bar,
+#t2i-top-2nd-split-bar {
+    width: var(--panel-gap, 7px) !important;
+}
+
+#t2i-top-2nd-split-bar {
+    margin-top: -8px;
+    position: relative;
+    z-index: 1;
+}
+
+/* Quick tools styling */
+.t2i-area-quicktools {
+    border: 1px solid var(--border-color) !important;
+    box-shadow: var(--shadow-md) !important;
+    margin-top: -20px;
+    padding: 0.5rem;
+}
+
+/* Image block styling */
+.image-block {
+    border: 1px solid transparent;
+    border-radius: 10px;
+    transition: all 0.3s ease;
+}
+
+.image-block:hover,
+.model-selected {
+    background: var(--box-selected-background) !important;
+    border: 1px solid var(--box-selected-border);
+    box-shadow: var(--shadow-md);
+}
+
+/* Category Headers and Expand/Retract Buttons */
+.auto-symbol {
+    display: inline-block;
+    text-align: center;
+    font-size: 20px;
+    width: 20px;
+    height: 17px;
+    margin-right: 10px;
+    position: relative;
+    top: 1.5px;
+    transition: background .2s;
+    color: var(--text) !important;
+}
+
+.input-group .header-label {
+    font-size: 15px;
+    color: var(--text);
+}
+
+/* Add this to ensure proper spacing in the input group */
+.input-group .input-group-content {
+    --spacing: 30px;
+    padding: 0;
+    max-width: unset;
+    margin-top: 10px;
+}
+
+.input-group .input-group-content > div {
+    padding: 5px 10px;
+    align-items: center;
+}
+
+.input-group .auto-input-name {
+    margin-right: 10px;
+    display: inline-flex;
+    flex-direction: row-reverse;
+    gap: 5px;
+    text-align: left;
+}
+
+/* Add styling for the quick buttons */
+.auto-input-qbutton {

```

Comment:
again, what is with these massive blocks of css? what are you changing? why? most of this looks expressly wrong to change this way, eg reassigning colors in css classes rather than variables.